### PR TITLE
Add schema details required for the MVP

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

File: src/cljc/witan/schema.cljc [hunk 1]
```diff
@@ -19,3 +20,101 @@
   "validation for /login"
   {(s/required-key :username) (s/both (length-greater 5) (is-an-email))
    (s/required-key :password) (length-greater 5)})
+
+(def IdType
+  s/Str)
```
Comment (elisehuard) [inline src/cljc/witan/schema.cljc]: are we going with uuid type or str?
still need to find out the advantages of one vs the other in terms of validation
(s/Uuid)

File: src/cljc/witan/schema.cljc [hunk 2]
```diff
@@ -19,3 +20,101 @@
...
+   (s/optional-key :description) s/Str
+   (s/required-key :forecast-id) IdType
+   (s/required-key :forecast-version) s/Int})
+
+(def Forecast
+  "Forecast"
+  {(s/required-key :series)        IdType
+   (s/required-key :name)          s/Str
+   (s/required-key :owner)         IdType
+   (s/required-key :version)       s/Int
```
Comment (elisehuard) [inline src/cljc/witan/schema.cljc:105]: would we need a separate structure ForecastVersion?
when we do a new version of forecast, we post the metadata of that forecast (probably in terms of input data sets in this version) linked to a ForecastId (just thinking aloud here)

File: src/cljc/witan/schema.cljc (hunk 1 above)
Comment (acron0) [inline src/cljc/witan/schema.cljc]: Which ever we choose needs to be both clj and cljs compatible (so the native JDK one won't be...)

Comment (elisehuard) [inline src/cljc/witan/schema.cljc]: true, didn't think about that.
But it seems the prismatic people did:
https://github.com/Prismatic/schema/blob/master/src/cljx/schema/coerce.cljx#L87

File: src/cljc/witan/schema.cljc (hunk 2 above)
Comment (acron0) [inline src/cljc/witan/schema.cljc:105]: I don't think so. `Forecast` represents a version of a forecast. When we POST a new forecast, it doesn't care which version inspired the change; the new version will always be the most recent version + 1.

Comment (elisehuard) [inline src/cljc/witan/schema.cljc:105]: OK, so series is the actual forecast id?

Comment (acron0) [inline src/cljc/witan/schema.cljc:105]: Yeah, worth renaming to just :id ?

Comment (elisehuard) [inline src/cljc/witan/schema.cljc:105]: or forecast_id? It's a tricky one, since the actual 'id' of this post is the version of this particular forecast

Comment (elisehuard) [inline src/cljc/witan/schema.cljc:105]: but then again #nitpicking so :shipit:

**Pull request merged**